HiP-DataStore/Controllers/QueryHelper.cs

```diff
@@ -51,13 +51,56 @@ public static class QueryHelper
                 : query.Where(x => x.Timestamp > timestamp.Value);
         }
 
-        public static IQueryable<T> FilterByUser<T>(this IQueryable<T> query, ContentStatus status, IIdentity user) where T : ContentBase
+        public static IQueryable<T> FilterByUser<T>(this IQueryable<T> query, ContentStatus status, IIdentity User) where T : ContentBase
         {
-            bool isAllowedGetAll = UserPermissions.IsAllowedToGetAll(user, status);
+            bool isAllowedGetAll = UserPermissions.IsAllowedToGetAll(User, status);
             return query.FilterIf(!isAllowedGetAll, x =>
-                        ((status == ContentStatus.All) && (x.Status == ContentStatus.Published)) || (x.UserId == user.GetUserIdentity()));
+                        ((status == ContentStatus.All) && (x.Status == ContentStatus.Published)) || (x.UserId == User.GetUserIdentity()));
         }
         /// <summary>
+        /// Returns all entries that are located within the radius, defined within the entry itself, around the given latitude and longitude.
+        /// If none or only one coordinate is given, all entries are returned.
+        /// </summary>
+        public static IQueryable<T> FilterByLocation<T>(this IQueryable<T> query, float? latitude, float? longitude) where T : ContentBase
+        {
+            List<int> excludedIds = new List<int>();
```

Comment:
You can shorten variable declarations using the [`var`-keyword](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/var), e.g.
```CSharp
var excludedIds = new List<int>()
```
As a rule of thumb and for consistency, please use `var` whenever possible.